jmx-metrics/src/test/groovy/io/opentelemetry/contrib/jmxmetrics/OtlpIntegrationTests.groovy

```diff
@@ -58,27 +60,21 @@ class OtlpIntegrationTests extends OtlpIntegrationTest {
         when: 'we examine the metric metadata'
         Metric metric = metrics.get(0)
         then: 'it is of the expected content'
-        metric.name == 'cassandra.storage.load'
-        metric.description == 'Size, in bytes, of the on disk data size this node manages'
-        metric.unit == 'By'
-        metric.hasSummary()
+        metric.name == 'cassandra.current_tasks'
+        metric.description == 'Number of tasks in queue with the given task status.'
+        metric.unit == '1'
+        metric.hasGauge()
 
         when: 'we examine the datapoints'
-        Summary datapoints = metric.summary
+        Gauge datapoints = metric.gauge
         then: 'they are of the expected size'
-        datapoints.dataPointsCount == 1
+        datapoints.dataPointsCount == 44
```

Comment:
I'm not sure if this is related to intermediary proto changes but the number of datapoints is now more manageable.  Would you mind adding more complete content validation?

Reply:
It may be preferable overall to keep the existing integration tests as they were and add another for this feature.

Reply:
moved to a separate test but had to modify the signature for `configureContainers` to allow me to pass in my own script.